### DISCUSSION TITLE
Force light mode

### DISCUSSION THREAD

**Discussion by VigHub** [skeletonlabs/skeleton#1737] [Q&A] [answered]
Hi all, 
Is there a way to force always to light mode? 
Without considering operating system setting preference and user's selected preference, and without display the `LightSwitch` component.
Thanks!

**Comment by Mahmoud-zino** [comment]
You can set the `$modeCurrent` store to true or use the method `setModeCurrent(true)`, this will force light mode.

You can read more about it at https://www.skeleton.dev/utilities/lightswitches under `Build Your Own` section.

**Reply by VigHub** [reply]
Thank you, I had already read that section in the documentation, but I didn't know where to put `setModeCurrent`, I tried in `+layout.svelte` file and got some cryptic error, so I posted my question.

**Comment by endigo9740** [comment, marked as answer]
@VigHub mahmoud's solution will work, but the simpler solution is to set `darkMode: 'class'` in the Tailwind config, per our normal instructions, then open `app.html` and remove the `dark` class from the `<html>` element at the top, if it's there at all. Then you should be good to go.

**Reply by VigHub** [reply]
Thank you, it worked as expected. Small change and it does the job.
